web/css/main.css

```
@@ -213,74 +339,1225 @@ body.pc-vertical-stretch {
+    @apply text-sm/6;
+    @apply font-semibold;
+    @apply text-pc-green;
+    @apply hover:text-pc-green-hover;
+}
+
+.pc-docs-link::after {
+    content: '';
+    width: 10px;
+    height: 10px;
+    flex-shrink: 0;
+    background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)